seqkit/2.3.1/Dockerfile

```diff
@@ -0,0 +1,47 @@
+# FROM defines the base docker image. This command has to come first in the file
+# The 'as' keyword lets you name the folowing stage. We use `app` for the production image
+FROM ubuntu:xenial as app
+
+# ARG sets environment variables during the build stage
+ARG SEQKIT_VER="2.3.1"
+
+# LABEL instructions tag the image with metadata that might be important to the user
+# Optional, but highly recommended
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="SeqKit"
+LABEL software.version=$SOFTWARENAME_VER
+LABEL description="SeqKit - a cross-platform and ultrafast toolkit for FASTA/Q file manipulation"
+LABEL website="https://github.com/shenwei356/seqkit"
+LABEL license="https://github.com/shenwei356/seqkit/blob/master/LICENSE"
+LABEL maintainer="Henry Kunerth"
+LABEL maintainer.email="henrykunerth@gmail.com"
+
+# Install dependences (update as needed)
+RUN apt-get update && apt-get install -y \
+	wget
+

```

Comment:
I love a tool that doesn't have a ton of dependencies! 

I do have a request to adjust this, however, because apt-get stores a lot of things, so your image will bigger than it needs to be.

Could you adjust this to something like
```
RUN apt-get update && apt-get install -y --no-install-recommends \
 wget && \
 apt-get autoclean && rm -rf /var/lib/apt/lists/*
```